draft-ietf-gnap-core-protocol.md

```diff
@@ -3261,12 +3215,106 @@ the token management URL as valid, since the end result is still the token
 not being usable.
 
 
-# Using Access Tokens {#use-access-token}
+# Securing Requests from the Client Instance {#secure-requests}
+
+In GNAP, the client instance secures its requests to the AS and RS by presenting an access
+token, presenting proof of a key that it possesses, or both an access token and
+key proof together.
+
+- When an access token is used with a key proof, this is a bound token request. This type of
+    request is used for calls to the RS as well as the AS during negotiation.
+- When a key proof is used with no access token, this is a non-authorized signed request. This
+    type of request is used for calls to the AS to initiate a negotiation.
+- When an access token is used with no key proof, this is a bearer token request. This type of
+    request is used only for calls to the RS, and only with access tokens that are
+    not bound to any key as described in {{response-token-single}}.
+- When neither an access token nor key proof are used, this is an unsecured request. This 
+    type of used only for calls to the RS during a discovery phase as
+    described in {{rs-request-without-token}}.
 
-The method the client instance uses to send an access token to the RS depends on the value of the
-"key" and "proof" parameters in [the access token response](#response-token-single).
+## Key Formats {#key-format}
 
-If the key value is the boolean `false`, the access token is a bearer token
+Several different places in GNAP require the presentation of key material 
+by value. Proof of this key material MUST be bound to a request, the nature of which varies with
+the location in the protocol the key is used. For a key used as part of a client instance's initial request
+in {{request-client}}, the key value is the client instance's public key, and
+proof of that key MUST be presented in that request. For a key used as part of an
+access token response in {{response-token-single}}, the proof of that key MUST
+be used when presenting the access token.
+
+A key presented by value MUST be a public key in at least one
+supported format. If a key is sent in multiple
+formats, all the key format values MUST be equivalent. Note that
```

Comment:
what do you mean exactly by equivalent? (just to be sure)

Reply:
It means they represent the same underlying key. So an RSA key in X509 format under `cert` can also be represented as a JWK under `jwk`, and either of these could be with or without CRT values, and all would be equivalent as defined by the RSA key type. If there's a reference for cryptographic key equivalence we should cite it here.